src/components/Footer/index.scss

```diff
@@ -61,22 +63,21 @@ $Logo-padding: 2px;
 }
 
 .Footer-tagline {
-
   color: $color-white;
 
   font-family: $font-family-serif;
   font-size: $size-small;
   line-height: $size-base;
-}
+};
 
 .Footer-logo {
-  padding: $Logo-padding;
+  padding: 2px;
 
   background-color: $color-yellow;
   color: $color-deep-blue;
 
   font-family: $font-family-mono;
-  font-size: $Logo-font-size-small;
+  font-size: 12px;
```

Comment:
`$size-xsmall`. Other than this, seems ready to go

Reply:
👍